sentry-ruby/lib/sentry/configuration.rb

```diff
@@ -201,6 +201,18 @@ def capture_exception_frame_locals=(value)
     # @return [String, nil]
     attr_accessor :sdk_debug_transport_log_file
 
+    # File path for DebugStructuredLogger to log events to. If not set, defaults to a temporary file.
+    # This is useful for debugging and testing structured logging.
+    # @return [String, nil]
+    attr_accessor :sdk_debug_structured_logger_log_file
```

Comment:
I do not like that we are polluting the main `configuration` namespace with this debug stuff, would prefer they be passed as init arguments to `DebugStructuredLogger.new` and instead of `structured_logger_class` we just have `structured_logger` where you can pass in your own instance.